pkgs/misc/wiki-tui/default.nix

```diff
@@ -19,25 +20,29 @@ rustPlatform.buildRustPackage rec {
     hash = "sha256-euyg4wYWYerYT3hKdOCjokx8lJldGN7E3PHimDgQy3U=";
```

Comment:
Also we no longer do repo = pname 😅

Reply:
Just out of curiosity, why to we no longer do this?

Reply:
https://github.com/NixOS/nixpkgs/issues/277994